Dynatrace/dynatrace-operator
# Adapt rotation interval for activeGate authToken

# Description

We should make sure that the ActiveGate Token deployed by Dynatrace Operator is going to be rotated at least 31 days before it actually expires. This is currently not the case because we rotate it after 30 days + dynakube reconciliation interval. This leads to an unexpected message in the UI

## How can this be tested?
 Create a Dynakube with an AG -> this will create a AuthToken -> Change the creationTimestamp -> Check if the token is rotated


## Checklist
